airflow/models/connection.py

```diff
@@ -66,43 +66,44 @@ class Connection(Base, LoggingMixin):
     _extra = Column('extra', String(5000))
 
     _types = [
-        ('docker', 'Docker Registry',),
+        ('docker', 'Docker Registry'),
         ('fs', 'File (path)'),
-        ('ftp', 'FTP',),
+        ('ftp', 'FTP'),
         ('google_cloud_platform', 'Google Cloud Platform'),
-        ('hdfs', 'HDFS',),
-        ('http', 'HTTP',),
-        ('pig_cli', 'Pig Client Wrapper',),
-        ('hive_cli', 'Hive Client Wrapper',),
-        ('hive_metastore', 'Hive Metastore Thrift',),
-        ('hiveserver2', 'Hive Server 2 Thrift',),
-        ('jdbc', 'Jdbc Connection',),
+        ('hdfs', 'HDFS'),
+        ('http', 'HTTP'),
+        ('pig_cli', 'Pig Client Wrapper'),
+        ('hive_cli', 'Hive Client Wrapper'),
+        ('hive_metastore', 'Hive Metastore Thrift'),
+        ('hiveserver2', 'Hive Server 2 Thrift'),
+        ('jdbc', 'JDBC Connection'),
+        ('odbc', 'ODBC Connection'),
```

Comment:
This is the only addition. `ODBC Connection`. The rest modifications are just to remove trailing `,` in the tuples to make it consistent.